srcs/server/server.cpp

```diff
@@ -84,17 +124,20 @@ namespace ft
 		{
 			httpRequest_pair_map_[new_client.client_id];
 		}
-		catch (const ft::Socket::connectionHangUp &deleted_client)
+		catch (const ft::Socket::closedConnection &deleted_client)
 		{
 			httpRequest_pair_map_.erase(deleted_client.client_id);
 		}
 		catch (const ft::Socket::NoRecieveMsg &e)
 		{
 			std::cerr << "no msg recieved" << std::endl;
 		}
```

Comment:
This is all the other parts of issue #49 as discussed in discord
https://discordapp.com/channels/1025574726548074527/1025574726548074531/1056797093764792353